Microsoft.Toolkit.Uwp.Services/Microsoft.Toolkit.Uwp.Services.csproj

```diff
@@ -11,7 +11,7 @@
     <AssemblyName>Microsoft.Toolkit.Uwp.Services</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.10586.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.15063.0</TargetPlatformVersion>
```

Comment:
This doesn't seem necessary since all the code is in the Microsoft.Toolkit.Uwp project